src/utils/telegram-logger.ts

```
@@ -0,0 +1,97 @@
+    message: string,
+    context?: string,
+    userId?: string,
+  ): string {
+    let emoji = 'ℹ️';
+    if (level === 'ERROR') emoji = '❌';
+    if (level === 'WARNING') emoji = '⚠️';
+    const timestamp = new Date().toISOString();
+
+    let header = `${emoji} <b>[${level}]</b> | <i>${timestamp}</i>`;
+    if (userId) header += ` | <code>User: ${userId}</code>`;
+    let body = `<pre>${this.escapeHtml(message)}</pre>`;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)